lib/Backend/Lower.cpp

```diff
@@ -25557,6 +25545,18 @@ Lowerer::InsertLoopTopLabel(IR::Instr * insertBeforeInstr)
     return loopTopLabel;
 }
 
+IR::Instr *
+Lowerer::AddBailoutToHelperCallInstr(IR::Instr * helperCallInstr, BailOutInfo * bailoutInfo, IR::BailOutKind  bailoutKind, IR::Instr * primaryBailoutInstr)
+{
+    helperCallInstr = helperCallInstr->ConvertToBailOutInstr(bailoutInfo, bailoutKind);
+    if (bailoutInfo->bailOutInstr == primaryBailoutInstr)
+    {
+        IR::Instr * instrShare = primaryBailoutInstr->ShareBailOut();
+        LowerBailTarget(instrShare);
```

Comment:
>instrShare [](start = 24, length = 10)

nit: No need for instrShare as we are not doing anything additional on it. Can directly pass primaryBailoutInstr->ShareBailOut() to LowerBailTarget

Reply:
instrShare just makes the shared bailout instruction stand out.